crev-lib/src/proofdb.rs

```diff
@@ -957,47 +963,43 @@ impl ProofDB {
             .map(|review| review.package.id.version.clone())
     }
 
-    fn record_url_from_to_field(&mut self, date: &DateTime<Utc>, to: &crev_data::PubId) {
-        self.url_by_id_secondary
-            .entry(to.id.clone())
-            .or_insert_with(|| TimestampedUrl {
-                value: to.url.clone(),
-                date: *date,
-            });
-    }
-
-    fn record_url_from_from_field(&mut self, date: &DateTime<Utc>, from: &crev_data::PubId) {
+    /// Record mapping between a PubId and a URL it declares, and trust it's correct only if it's been fetched from the same URL
```

Comment:
I'm not sure if I'm supposed to re-review already or not. But in case I am - it doesn't make sense to check url source here. If the id signed a proof and provided its url, it doesn't matter if it was from their own repo, someone elses repo, or anywhere else - it is a verified information.